<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>posture_window</class>
 <widget class="QMainWindow" name="posture_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	background-color: rgb(66, 66, 66);
	border: none;
	background: none;
	color: #fff;
}
QPushButton, QLineEdit, QCheckBox{
	background-color:rgba(10,16,27,150);
	padding: 7px;
	border-radius: 5px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>699</width>
     <height>411</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item alignment="Qt::AlignTop">
     <widget class="QFrame" name="header_frame">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 67, 98);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item alignment="Qt::AlignLeft">
        <widget class="QFrame" name="header_left_frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="return_btn">
            <property name="font">
             <font>
              <family>Lucida Sans</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <italic>true</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Main Menu</string>
            </property>
            <property name="icon">
             <iconset resource="icons/icons.qrc">
              <normaloff>:/icons/arrow-left.svg</normaloff>:/icons/arrow-left.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="header_center_frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="icons/icons.qrc">:/icons/wheelchair.png</pixmap>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="main_title">
            <property name="font">
             <font>
              <family>Lucida Sans</family>
              <pointsize>26</pointsize>
              <weight>75</weight>
              <italic>true</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Smart wheelchair&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="header_right_frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="main_frame">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>325</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(66, 66, 66);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QFrame" name="pages">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QScrollArea" name="scrollArea">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>0</y>
            <width>661</width>
            <height>311</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>661</width>
             <height>311</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <property name="spacing">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QStackedWidget" name="stackedWidget">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QWidget" name="page_3">
               <widget class="QLabel" name="label_2">
                <property name="geometry">
                 <rect>
                  <x>20</x>
                  <y>10</y>
                  <width>621</width>
                  <height>91</height>
                 </rect>
                </property>
                <property name="font">
                 <font>
                  <family>Lucida Sans</family>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Welcome to monitoring your system!&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;You can turn on the system clicking on the button&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
               <widget class="QPushButton" name="start_wizard_btn">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="geometry">
                 <rect>
                  <x>210</x>
                  <y>130</y>
                  <width>181</width>
                  <height>141</height>
                 </rect>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="icons/icons.qrc">
                  <normaloff>:/icons/power-on.png</normaloff>:/icons/power-on.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>128</width>
                  <height>128</height>
                 </size>
                </property>
               </widget>
              </widget>
              <widget class="QWidget" name="page">
               <widget class="QLabel" name="label_5">
                <property name="geometry">
                 <rect>
                  <x>10</x>
                  <y>10</y>
                  <width>331</width>
                  <height>281</height>
                 </rect>
                </property>
                <property name="font">
                 <font>
                  <family>Lucida Sans</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="autoFillBackground">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;Please click on the buttons in the direction you want to operate the system. &lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;You also have a velocity bar to change the speed of the wheelchair.&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;You have an emergency button to stop the system.&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;To come back and start the controller next page have a return button for that.&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;Let's go to operate the wheelchair!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="scaledContents">
                 <bool>true</bool>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item alignment="Qt::AlignBottom">
     <widget class="QFrame" name="footer_frame">
      <property name="styleSheet">
       <string notr="true">
background-color: rgb(66, 66, 66);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="footer_left_frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item alignment="Qt::AlignLeft">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Version 1.0 Copyright AXGC&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="footer_right_frame">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(66,66, 66);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item alignment="Qt::AlignRight">
           <widget class="QPushButton" name="info_btn">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="icons/icons.qrc">
              <normaloff>:/icons/help-circle.svg</normaloff>:/icons/help-circle.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>28</width>
              <height>28</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item alignment="Qt::AlignBottom">
        <widget class="QFrame" name="size_grip_2">
         <property name="minimumSize">
          <size>
           <width>10</width>
           <height>10</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>10</width>
           <height>10</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
